frontend/src/lib/components/hedgehogs.tsx

```diff
@@ -140,3 +141,6 @@ export const MicrophoneHog = (props: HedgehogProps): JSX.Element => {
 export const PhonePairHogs = (props: HedgehogProps): JSX.Element => {
     return <SquaredHedgehog src={phonePairHogs} {...props} />
 }
+export const BuilderHogWavingSmiling = (props: HedgehogProps): JSX.Element => {
+    return <SquaredHedgehog src={builderHogWavingSmiling} {...props} />
+}
```

Comment:
Do we not already have this one in there? Prob remove since not using it anyway.